docs/azure/sdk/snippets/aspnetcore-guidance/MinApiSample/Program.cs

```diff
@@ -0,0 +1,66 @@
+using Azure.Identity;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Extensions.Azure;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+builder.Services.AddAzureClients(clientBuilder =>
+{
+    // Register clients using a config file section
+    clientBuilder.AddSecretClient(
+        builder.Configuration.GetSection("KeyVault"));
+
+    clientBuilder.AddBlobServiceClient(
+        builder.Configuration.GetSection("Storage"));
+
+    // Register clients using a specific config key-value pair
+    clientBuilder.AddServiceBusClientWithNamespace(
+        builder.Configuration["ServiceBus:Namespace"]);
+
+    clientBuilder.UseCredential(new DefaultAzureCredential());
+
+    // Set up any default settings
+    clientBuilder.ConfigureDefaults(
+        builder.Configuration.GetSection("AzureDefaults"));
+});
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+
+app.UseHttpsRedirection();
+app.MapGet("/reports", async (BlobServiceClient blobServiceClient) =>
+{
+    // Use the blob client
+    BlobContainerClient containerClient
+        = blobServiceClient.GetBlobContainerClient("reports");
+
+    var reports = new List<BlobItem>();
```

Comment:
Make consistent with the Blazor code sample
```suggestion
    List<BlobItem> reports = new();
```